website/www/site/content/en/contribute/release-guide.md

```diff
@@ -583,189 +582,44 @@ For this step, we recommend you using automation script to create a RC, but you
   1. Stage source release into dist.apache.org dev [repo](https://dist.apache.org/repos/dist/dev/beam/).
   1. Stage,sign and hash python binaries into dist.apache.ord dev repo python dir
   1. Stage SDK docker images to [docker hub Apache organization](https://hub.docker.com/search?q=apache%2Fbeam&type=image).
-  1. Create a PR to update beam and beam-site, changes includes:
+  1. Create a PR to update beam-site, changes includes:
      * Copy python doc into beam-site
      * Copy java doc into beam-site
-     * Update release version into [_config.yml](https://github.com/apache/beam/blob/master/website/_config.yml).
      
 #### Tasks you need to do manually
-  1. Add new release into `website/src/get-started/downloads.md`.
-  1. Update last release download links in `website/src/get-started/downloads.md`.
-  1. Update `website/src/.htaccess` to redirect to the new version.
-  1. Build and stage python wheels.
+  1. Verify the script worked.
+      1. Verify that the source and Python binaries are present in [dist.apache.org](https://dist.apache.org/repos/dist/dev/beam).
+      1. Verify Docker images are published. How to find images:
+          1. Visit [https://hub.docker.com/u/apache](https://hub.docker.com/search?q=apache%2Fbeam&type=image)
+          2. Visit each repository and navigate to *tags* tab.
+          3. Verify images are pushed with tags: ${RELEASE}_rc{RC_NUM}
+      1. Verify that third party licenses are included in Docker containers by logging in to the images.
+          - For Python SDK images, there should be around 80 ~ 100 dependencies.
+          Please note that dependencies for the SDKs with different Python versions vary.
+          Need to verify all Python images by replacing `${ver}` with each supported Python version `X.Y`.
+          ```
+          docker run -it --entrypoint=/bin/bash apache/beam_python${ver}_sdk:${RELEASE}_rc{RC_NUM}
+          ls -al /opt/apache/beam/third_party_licenses/ | wc -l
+          ```
+          - For Java SDK images, there should be around 1400 dependencies.
+          ```
+          docker run -it --entrypoint=/bin/bash apache/beam_java_sdk:${RELEASE}_rc{RC_NUM}
+          ls -al /opt/apache/beam/third_party_licenses/ | wc -l
+          ```
   1. Publish staging artifacts
-      1. Go to the staging repo to close the staging repository on [Apache Nexus](https://repository.apache.org/#stagingRepositories). 
+      1. Go to the staging repo to close the staging repository on [Apache Nexus](https://repository.apache.org/#stagingRepositories).
       1. When prompted for a description, enter “Apache Beam, version X, release candidate Y”.
-
-
-### (Alternative) Run all steps manually
```

Comment:
Removing this big chunk of text because?

Reply:
Because it's mostly an exact copy of build_release_candidate.sh. Any part of this block that wasn't part of build_release_candidate.sh was moved into the "manual steps" section.